internal/wasi_snapshot_preview1/wasi.go

```diff
@@ -93,3 +99,19 @@ var errnoToString = [...]string{
 	"EXDEV",
 	"ENOTCAPABLE",
 }
+
+func ValueLoggers(fnd api.FunctionDefinition) (pLoggers []logging.ParamLogger, rLoggers []logging.ResultLogger) {
```

Comment:
this is the start, where special-casing is in the package defining the functions, not in the logging package. This is possible due to the logging types introduced in this change.